toolkit/templates/big-stat.html

```diff
@@ -0,0 +1,7 @@
+{% macro big_stat(stat_value, stat_name, id=None) -%}
```

Comment:
Does this need to be a macro? If you dropped the first and last lines and just left the html it would still work? And you examples page in pages_builder could look like:
```
examples:
  -
    stat_value: 12
    stat_name: started applications
    id: started-applications
  -
    stat_value: 1
    stat_name: completed applications
    id: completed-applications

```